### ISSUE TITLE
Why is my FID so high using the FID code provided there?

### ISSUE THREAD

**Issue by furukawayuan-Yao** [MinfengZhu/DM-GAN#18]
I  use **'DM-GAN/eval/FID/find_score.py'** to calculate the FID  of  **AttnGAN** on COCO dataset. Before that I set the **B_VALIDATION = TRUE** ,changed the **BATCH_SIZE=50**, then trained my model 120 epoches to produce evaluation image. I didn't change anywhere in **find_score.py** . However, I got  **FID=62.00581731786599**！Is it because the coco_val.npz model I used which can't be used on AttnGAN. Has anyone ever been in this situation and solved it? 😔

**Comment by MinfengZhu** [comment]
How many images have you generated? We generate 30,000 images to conduct all evaluations.

We compute coco_val.npz using the validation set. It can be used to evaluate AttnGAN. You can also measure FID using images from the validation set directly.

**Comment by furukawayuan-Yao** [comment]
> How many images have you generated? We generate 30,000 images to conduct all evaluations.
> 
> We compute coco_val.npz using the validation set. It can be used to evaluate AttnGAN. You can also measure FID using images from the validation set directly.



> How many images have you generated? We generate 30,000 images to conduct all evaluations.
> 
> We compute coco_val.npz using the validation set. It can be used to evaluate AttnGAN. You can also measure FID using images from the validation set directly.

I've generated 40624 images. But now I may figure out the reason why my FID so high. I set the branch num 2 and the size of image generated is 128*128.  Can you tell me the **BATCH_SIZE**  you choose for reproducing AttnGAN ? Thanks a lot!(*^_^*)

**Comment by MinfengZhu** [comment]
We use the pre-trained model to evaluate AttnGAN directly. If you want to train AttnGAN from scratch, you can use the default setting. We also found that batch size may affect the performance, especially R-precision. Because batch size controls the number of negative samples in DAMSM loss.
